src/util/generatePreview.js

```diff
@@ -39,6 +39,63 @@ const errorHandlerScript = `(${function() {
   };
 }.toString()}());`;
 
+const elementHighlighterScript = `(${function() {
+  let selector = '';
+  window.addEventListener('message', (message) => {
+    const data = JSON.parse(message.data);
+    const {selector: {selector: selectorFromMessage}} = data;
+    selector = selectorFromMessage;
+    removeCovers();
+    generateCovers();
+  });
+
+  window.addEventListener('resize', () => {
+    removeCovers();
+    generateCovers();
+  });
+
+  window.addEventListener('scroll', () => {
+    removeCovers();
+    generateCovers();
+  });
+
+  function removeCovers() {
+    const highlighterElements =
+    window.document.querySelectorAll('.popcode_HIGHLIGHTER');
+    if (highlighterElements.length !== 0) {
+      highlighterElements.forEach((highlighterElement) => {
+        highlighterElement.remove();
+      });
+    }
+  }
+
+  function generateCovers() {
+    if (selector !== '') {
+      const elements =
+      window.document.querySelectorAll(selector);
+      elements.forEach((element) => {
+        const rect = element.getBoundingClientRect();
+        const cover =
+        window.document.documentElement.appendChild(
+          window.document.createElement('div'));
+        cover.className = 'popcode_HIGHLIGHTER';
+        cover.style.position = 'fixed';
+        cover.style.zIndex = 10000000000;
+        cover.style.border = '1px solid rgba(238, 238, 48, 0.5)';
+        cover.style.boxSizing = 'border-box';
+        cover.style.boxShadow = '0 0 20px rgba(0,0,0,0.5)';
+        cover.style.pointerEvents = 'none';
+        cover.style.color = 'black';
+        cover.style.background = 'transparent';
```

Comment:
I think it would make sense to put all these style declarations in a stylesheet in the preview document. (Obviously the below, being dynamically generated, should stay here)

Reply:
Sounds cleaner! Where does it make sense to store the CSS source for this?

Reply:
Probably in the `templates` directory, and then just import it as plain text?